CookBook/src/app/auth/auth.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+    <div class="col-xs-12 col-md-6 col-md-offset-3">
+        <div class="alert alert-danger" *ngIf="error">
+            <p>{{ error }}</p>
+        </div>
+        <div *ngIf="isLoading" style="text-align: center;">
+            <app-loading-spinner></app-loading-spinner>
+        </div>
+        <form #authForm="ngForm" (ngSubmit)="onSubmit(authForm)" *ngIf="!isLoading">
```

Comment:
Mientra que esto funciona y esta bien hecho, los Template Driven forms traen problemas al momento de complejizarse e implementar validaciones mas complejas. Trata de rehacer esto con Reactive Forms.

Reply:
fix: Auth component to reactive form subido

Reply:
Perfecto